libs-scala/db-utils/src/main/scala/com/digitalasset/http/dbbackend/DBConfig.scala

```diff
@@ -105,8 +107,9 @@ object JdbcConfig
     with StrictLogging {
 
   final val MinIdle = 8
-  final val IdleTimeout = 10000L // ms, minimum according to log, defaults to 600s
-  final val ConnectionTimeout = 5000L
+  final val IdleTimeout =
+    FiniteDuration(10000, TimeUnit.MILLISECONDS) // ms, minimum according to log, defaults to 600s
```

Comment:
```suggestion
    FiniteDuration(10000, TimeUnit.MILLISECONDS) // minimum according to log, defaults to 600s
```
Good call making this a `Duration`, now the unit in the comment is no longer needed. I'm also not sure of what the rest of the comment means.

Reply:
Its again a copy paste, I am not completely sure either, happy to get rid of the comment